bibliography.bib

```diff
@@ -1,3 +1,94 @@
+
+@ONLINE{hazelcast,
+	TITLE={Hazelcast Jet},
+	ORGANIZATION={{Hazelcast, Inc.}},
+	URLDATE= {2019-03-27},
+	URL= {https://hazelcast.com/products/jet/},
+}
+
+@INPROCEEDINGS{tasci2015, 
+    AUTHOR={S. {Tasci} and M. {Demirbas}}, 
+    BOOKTITLE={2015 IEEE International Conference on Big Data (Big Data)}, 
+    TITLE={Employing in-memory data grids for distributed graph processing}, 
+    YEAR={2015}, 
+    PAGES={1856-1864}, 
+    KEYWORDS={Big Data;data structures;distributed processing;graph theory;in-memory data grids;distributed graph processing;IMDG;big data;RAM;unit tests;core distributed operations;data structures;performance tradeoffs;large-scale graph processing;batch-mode graph processing;online graph processing;social networks;Web services applications;Distributed databases;Data structures;Servers;Space exploration;Scalability;Big data}, 
+    DOI={10.1109/BigData.2015.7363959}, 
+    MONTH={10},}
+
+@ONLINE{hazelcastmanual,
+	TITLE={Hazelcast IMDG Reference Manual},
+	ORGANIZATION={{Hazelcast, Inc.}},
+	URLDATE= {2019-03-27},
+	URL= {https://docs.hazelcast.org/docs/3.11.2/manual/html-single/index.html#hazelcast-imdg-reference-manual},
+}
+
+@BOOK{johns2015,
+	AUTHOR = {M. {Johns}},
+	YEAR = {2015},
+	MONTH={07},
+	TITLE = {Getting Started with Hazelcast - Second Edition - },
+	EDITION = {2},
+	ISBN = {978-1-785-28533-2},
+	PUBLISHER = {Packt Publishing},
+	ADDRESS = {Birmingham},
+}
+
+@ONLINE{hazelcastnosql,
+	TITLE={In-Memory NoSQL with Hazelcast IMDG},
+	ORGANIZATION={{Hazelcast, Inc.}},
+	URLDATE= {2019-03-27},
+	URL= {https://hazelcast.org/use-cases/in-memory-nosql/},
+}
+
+@ONLINE{hazelcastredis,
+	TITLE={Redis Replacement},
+	ORGANIZATION={{Hazelcast, Inc.}},
+	URLDATE= {2019-03-27},
+	URL= {https://hazelcast.org/use-cases/redis-replacement/},
+}
+
+@BOOK{keyValueIntro,
+	AUTHOR = {T. {Kudraß}},
+	YEAR = {2015},
+	MONTH={03},
+	TITLE = {Taschenbuch Datenbanken},
+	ISBN = {9783446440265},
+	PUBLISHER = {Packt Publishing},
+	PAGES={369-373}
+}
+
+@ONLINE{keyValueUsecase,
+	TITLE={From RDBMS to Key-Value Store: Data Modeling Techniques},
+	AUTHOR={Wishmitha S. Mendis},
+	URLDATE={2017-04-02},
+	URL={https://medium.com/@wishmithasmendis/from-rdbms-to-key-value-store-data-modeling-techniques-a2874906bc46}
+}
+
+@ONLINE{hazelcastCAP,
+	TITLE={What CAP Theorem Means to a Business Leader},
+	ORGANIZATION={{Hazelcast, Inc.}},
+	URLDATE= {2019-04-01},
+	URL= {https://hazelcast.com/blog/what-cap-theorem-means-to-a-business-leader/},
+}
+
+@ONLINE{hazelcastCP,
+	TITLE={Jepsen Analysis on Hazelcast 3.8.3},
+    AUTHOR ={Greg Luck},
+	ORGANIZATION={{Hazelcast, Inc.}},
+	URLDATE= {2019-04-01},
+	URL= {https://hazelcast.com/blog/jepsen-analysis-hazelcast-3-8-3/},
+}
+
+@ONLINE{hazelcast:hazelcastDownload,
+    title={Hazelcast Downloads},
+	organization={{Hazelcast, Inc.}},
+	year = {},
+	month = {},
+	urldate= {2019-04-05},
+	url= {https://hazelcast.org/download/},
+}
+
 % Encoding: UTF-8
```

Comment:
Magic comments should always be at top of the file.

Reply:
I guess that depends on the editor, vim can read magic lines at the end of a file.

Reply:
But I don't think we need it here.

Reply:
This is still not resolved.